bugbug/bug_features.py

```diff
@@ -463,44 +463,46 @@ def __call__(self, bug, **kwargs):
 
 class couple_common_whiteboard_keywords(couple_bug_feature):
```

Comment:
I'd define a new feature (`couple_number_of_common_whiteboard_keywords`) instead of changing this one

Reply:
Or maybe not, if we are not using the old in any other place